.ado/templates/apple-xcode-build.yml

```diff
@@ -12,9 +12,8 @@ parameters:
 steps:
   # Install any pods necessary for the project
   - bash: |
-      echo "Running: cd apps/apple/$platform_directory && pod install" &&
-      cd apps/apple/$platform_directory &&
-      pod install
+      echo "Running: pod install --project-directory=$platform_directory/" &&
+      pod install --project-directory=$platform_directory/
```

Comment:
Looking at the CI build failure, it looks like you need the full path and this should be 
`pod install --project-directory=apps/apple/FluentUITester/$platform_directory`